framework/include/interfaces/Coupleable.h

```diff
@@ -818,6 +818,16 @@ class Coupleable
   virtual const ArrayVariableGradient & coupledArrayGradientOlder(const std::string & var_name,
                                                                   unsigned int comp = 0) const;
 
+  /**
+   * Retun gradient of a coupled array variable's time derivative
+   * @param var_name Name of coupled vector variable
+   * @param comp Component number for vector of coupled vector variables
+   * @return Reference to a ArrayVariableGradient containing the time derivative of the gradient of
```

Comment:
Is it the time derivative of the gradient, or the gradient of the time derivative? Above it's the latter but here it's the former